cluster/v1beta1/types_placement.go

```diff
@@ -94,6 +94,42 @@ type PlacementSpec struct {
 	// certain taints to be selected by placements with matching tolerations.
 	// +optional
 	Tolerations []Toleration `json:"tolerations,omitempty"`
+
+	// Decision Strategy divide the created placement decision to groups and define number of clusters per decision group.
+	// +optional
+	DecisionStrategy DecisionStrategy `json:"decisionStrategy,omitempty"`
+}
+
+// Subset of the created placementDecision with label.
+type DecisionGroup struct {
+	// Label to be added to the created placement Decisions
+	// +kubebuilder:validation:Required
+	// +required
+	Label string `json:"labels,omitempty"`
```

Comment:
what does the value of this field look like

Reply:
I see, label may cause confusion. May be better to change the field to name or groupName as we don't want the user to add multiple labels.